classes/PrestaShopCollection.php

```diff
@@ -157,7 +157,7 @@ public function join($association, $on = '', $type = null)
      * @param mixed $value
      * @param string $method where|having
      *
-     * @return PrestaShopCollection
+     * @return $this
```

Comment:
```suggestion
     * @return static
```

Don't we use `static` in the PHPDoc usually instead of `$this`

Reply:
It's the same thing.

Reply:
I know it's just a matter of convention like coding style
I know @PierreRambaud is picky about this kind of stuff, we should uniformize our use of fluent functions